### PR TITLE
Pass contact-summary to forms context.

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

Also adds Selectors unit tests.

medic/cht-core#7022

# Code review checklist

- [x] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [x] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [x] Tested: Unit and/or e2e where appropriate
- [x] Internationalised: All user facing text
- [x] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
hey @latin-panda . Could you please have a look? Thanks!

**Comment by dianabarsan** [comment]
@latin-panda as I was re-reading this, I realized a small performance improvement was possible, so I've added pushed that change.

File: webapp/tests/karma/ts/selectors/index.spec.ts [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+    loaded: 'are tasks loaded?',
+    selected: { _id: 'selected task' },
+  },
+};
+const clonedState = cloneDeep(state);
+
+describe('Selectors', () => {
+  afterEach(() => {
+    // state is never mutated!!!
+    expect(state).to.deep.equal(clonedState);
```
Comment (latin-panda) [inline webapp/tests/karma/ts/selectors/index.spec.ts:109]: ❤️

**Review by latin-panda — APPROVED** [review]
Nice job!! Love the new unit test!

**Pull request merged**